algorithms/src/snark/varuna/tests.rs

```diff
@@ -76,7 +76,7 @@ mod varuna {
                         assert!($snark_inst::verify(universal_verifier, &fs_parameters, &index_vk, public_inputs, &proof).unwrap());
                         println!("Called verifier");
                         eprintln!("\nShould not verify (i.e. verifier messages should print below):");
-                        assert!(!$snark_inst::verify(universal_verifier, &fs_parameters, &index_vk, [random, random], &proof).unwrap());
+                        assert!(!$snark_inst::verify(universal_verifier, &fs_parameters, &index_vk, [random], &proof).unwrap());
```

Comment:
Why was this changed?

Reply:
The new test and code introduced in this PR succesfully prevents too many inputs from being passed in. We used to pass in two public inputs here, even though the circuit only expects one public input.

For extra context: this is a negative test, so we're passing a random public input and expect it to fail.